device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,46 @@
+{%- set default_cable = '5m' %}
```

Comment:
Hi Joyesh, We need rebase here. Seems something got messed up here, these files should not be in this diff. I remember I approve this PR, but kindly fix this last comment and I'll approve  again, thanks a lot